ExampleApp.HttpServices/Program.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Dwolla.Client.HttpServices;
+using Dwolla.Client.HttpServices.Architecture;
+using ExampleApp.HttpServices.Tasks;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using static System.Console;
+
+namespace ExampleApp.HttpServices
+{
+	public class Program
+	{
+		private static void Main()
+		{
+			Environment.SetEnvironmentVariable("DWOLLA_APP_KEY", "zn0ErJ9xvfPYowr2Oprz7Ly5SRKvWYw7wWE86g09GGAqWA81nq");
```

Comment:
Don't forget to remove these. You might want to just set them in actual env vars on your computer so you don't have to keep pasting and removing them from code. Let me know if you want help with that